lab-teddy/README.md

```diff
@@ -0,0 +1,37 @@
+# Description:
+
+### For this project we needed to re-factor the code that we wrote for lab 08 so that we could have write the files into a new data file. It was kinda hard to do due to having to figure out what to take out and what to keep.
```

Comment:
So good job including a read me. Though describe more about your endpoints and how to use them